### DISCUSSION TITLE
Error calling Deepgram (Nova)

### DISCUSSION THREAD

**Discussion by alkorin452** [deepgram/community#361] [General help]
### Which Deepgram product are you using?

Nova

### Details

Hi all,

We have been using Deepgram's transcription service using Nova on prerecorded audio for quite a while with no issues. Today we experienced failures when calling transcription (using the node SDK). We are getting empty results in the response (example below) This is affecting both our development and production environment. There have been no changes on our side for ~ 1 month. Is there an ongoing outage with this service?
```
 "results": {
        "channels": [
            {
                "alternatives": [
                    {
                        "transcript": "",
                        "confidence": 0,
                        "words": []
                    }
                ]
            }
        ]
    }
```

### If you are making a request to the Deepgram API and have a request ID, please paste it below:
RequestId: 7780fdc8-b83f-42d8-84a1-97ebad969bdb

### If possible, please attach your code or paste it into the text box.

```const deepgramApiKey = await getDeepgramApiKey();
    const mimetype = "audio/wav";
    const deepgram = new Deepgram(deepgramApiKey);
    await retry(
    async () => await deepgram.transcription.preRecorded(
        { buffer: fs.readFileSync(filePath), mimetype },
        {
          punctuate: true,
          language: "en",
          model: "nova",
          callback: `${process.env.storeTranscriptUrl}?assetId=${assetId}&auth=${auth0AccessToken}`,
        }
      ), 3
    )
  } catch (err) {
    throw Error(`Error uploading transcript to deepgram, ${err.message}  `)
  }```

### If possible, please attach an example audio file to reproduce the issue.

_No response_

**Comment by jpvajda** [comment]
@alkorin452  what version of the Node SDK are you currently using? I'll share this post with @lukeocodes  who's one of our lead engineers on the Node SDK as he might have some ideas.
